draft-ietf-acme-acme.md

```diff
@@ -2726,6 +2726,10 @@ within their trusted network and use these resolvers both for both CAA record
 lookups and all record lookups in furtherance of a challenge scheme (A, AAAA,
 TXT, etc.).
 
+Further, an ACME-based CA can use the Certification Authority Authorization
```

Comment:
Do we have consensus on using the non-normative "can" here, instead of e.g. SHOULD? CAA is very much security-relevant.